internal/manifests/build.go

```diff
@@ -52,6 +52,16 @@ func BuildAll(opts Options) ([]client.Object, error) {
 	res = append(res, queryFrontendObjs...)
 	res = append(res, BuildLokiGossipRingService(opts.Name))
 
+	if opts.Flags.EnableLokiStackGateway {
+		gatewayObjects, gatewaySHA1, err := BuildLokiStackGateway(opts)
+		if err != nil {
+			return nil, err
+		}
+		opts.GatewaySHA1 = gatewaySHA1
```

Comment:
The Gateway configmap SHA1 is not required in the general options struct. It is used only by the gateway deployment.